source/App.cpp

```diff
@@ -87,34 +86,42 @@ void App::onInit() {
 	updateMouseSensitivity();			// Update (apply) mouse sensitivity
 	updateSessionDropDown();			// Update the session drop down to remove already completed sessions
 	updateSessionPress();				// Update session to create results file/start collection
+
+	setFrameDuration(m_wallClockTargetDuration, REAL_TIME);
```

Comment:
Yes, this is just needed to set GApp::REAL_TIME. I don't want to touch the initial m_wallClockTargetDuration value but G3D doens't have any way to do this individually. I removed the setFrameDuration call from updateParameters() in the second patch because it breaks testing, which uses a fixed timestep for frame perfect tests.

Reply:
I added some comments to these areas to clarify.